maptivist/src/main/webapp/script.js

```diff
@@ -191,8 +191,15 @@ function drawMarker({id, latitude, longitude, title, description, links, categor
 /**
 * Adds an infowindow based on the marker creation form
  */
+
+function createInfowindow(position, marker) {
+  //Set info window content from form
```

Comment:
You shouldn't need the createInfowindow function at all, it's not called anywhere so that's why it's not affecting anything. Could you delete it?

Reply:
+1